### PR TITLE
RHS Chdkz AI template

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information: 
Adds a complete AI template for the ChDKZ faction from the RHS modpack.
Adds a ChDKZ flag map marker and corresponding texture.

The ChDKZ as a whole would feel like a mix of the CDF and the AFRF, but utilizing older and more mixed equipment.
    

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Testing shouldn't be required, it runs with no errors that could be caught by a rpt.
However trying out the template on the test server to assess and adjust balance may be desirable.

********************************************************
Notes:
The ChDKZ while not depicted as the most organised in Arma2 are still able to field large amounts of troops and armor, to a degree arguably beyond that of a paramilitary and able to nearly conquer a whole country without overt outside intervention. That and the fact that they're a good bad guy is why i'm proposing them as an faction.

### CODE REVIEW

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 1]
```diff
@@ -0,0 +1,1028 @@
...
+["vehiclesTransportBoats", ["O_G_Boat_Transport_01_F"]] call _fnc_saveToTemplate;
+["vehiclesGunBoats", ["rhsusf_mkvsoc"]] call _fnc_saveToTemplate;
+["vehiclesAmphibious", ["rhsgref_ins_btr60","rhsgref_ins_btr70", "rhsgref_BRDM2_ins","rhsgref_BRDM2_ins", "rhsgref_ins_bmp2","rhsgref_ins_bmp1"]] call _fnc_saveToTemplate;
+
+["vehiclesPlanesCAS", ["RHS_Su25SM_vvsc"]] call _fnc_saveToTemplate;
+["vehiclesPlanesAA", ["rhs_mig29s_vvs","rhs_mig29sm_vvs"]] call _fnc_saveToTemplate;
+["vehiclesPlanesTransport", []] call _fnc_saveToTemplate;
+
+["vehiclesHelisLight", ["RHS_Mi8T_vvsc"]] call _fnc_saveToTemplate;
+["vehiclesHelisTransport", ["RHS_Mi8mt_vvsc","RHS_Mi8mt_vvsc", "RHS_Mi24Vt_vvsc"]] call _fnc_saveToTemplate; //Mi8mt has pk's, Mi24Vt has 12.7 turret only
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:64]: I think the Mi24Vt is acceptable as it doesn't have real pylons.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 2]
```diff
@@ -0,0 +1,1028 @@
...
+["vehiclesHelisLightAttack", ["RHS_Mi8MTV3_vvsc"]] call _fnc_saveToTemplate;
+["vehiclesHelisAttack", ["RHS_Mi8MTV3_vvsc","RHS_Mi8MTV3_heavy_vvsc", "RHS_Mi24P_vvsc", "RHS_Mi24V_vvsc"]] call _fnc_saveToTemplate;
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:66]: The MTV3s are just light attack. Rockets + MGs. In the AFRF template I even put the 24P in light attack, as they have a lot of better helis and the AI are probably useless with the fixed gun. You can set it as a cheap attack heli for variety and tiering. Cost maybe 170 or so.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 3]
```diff
@@ -0,0 +1,1028 @@
...
+["vehiclesAmmoTrucks", ["rhsgref_ins_gaz66_ammo"]] call _fnc_saveToTemplate;
+["vehiclesRepairTrucks", ["rhsgref_ins_gaz66_repair","rhsgref_ins_ural_repair"]] call _fnc_saveToTemplate;
+["vehiclesFuelTrucks", ["rhsgref_ins_kraz255b1_fuel"]] call _fnc_saveToTemplate;
+["vehiclesMedical", ["rhsgref_ins_gaz66_ap2"]] call _fnc_saveToTemplate;
+
+["vehiclesLightAPCs", ["rhsgref_BRDM2_ins","rhsgref_BRDM2_ins", "rhsgref_BRDM2UM_ins", "rhsgref_BRDM2_HQ_ins"]] call _fnc_saveToTemplate;            //this line determines light APCs
+["vehiclesAPCs", ["rhsgref_ins_btr60","rhsgref_ins_btr70"]] call _fnc_saveToTemplate;
+["vehiclesIFVs", ["rhsgref_ins_bmd2","rhsgref_ins_bmd1p","rhsgref_ins_bmd1","rhsgref_ins_bmp2k","rhsgref_ins_bmp2d","rhsgref_ins_bmp2e","rhsgref_ins_bmp2","rhsgref_ins_bmp1p","rhsgref_ins_bmp1k","rhsgref_ins_bmp1d","rhsgref_ins_bmp1"]] call _fnc_saveToTemplate;                //this line determines IFVs
+["vehiclesTanks", ["rhsgref_ins_t72ba","rhsgref_ins_t72bb","rhsgref_ins_t72bc"]] call _fnc_saveToTemplate;
+["vehiclesAA", ["rhsgref_ins_ural_Zu23","rhsgref_ins_gaz66_zu23", "rhsgref_ins_zsu234"]] call _fnc_saveToTemplate;
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:52]: Bump the zu trucks down to light armed. Anything in vehiclesAA needs to be armoured, unless you're desperate.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 4]
```diff
@@ -0,0 +1,1028 @@
...
+"rhsgref_ins_gaz66o_flat","rhsgref_ins_gaz66_flat",
+"rhsgref_ins_zil131_flatbed_cover","rhsgref_ins_zil131_flatbed",
+"rhsgref_ins_kraz255b1_flatbed"
+]] call _fnc_saveToTemplate;
+["vehiclesAmmoTrucks", ["rhsgref_ins_gaz66_ammo"]] call _fnc_saveToTemplate;
+["vehiclesRepairTrucks", ["rhsgref_ins_gaz66_repair","rhsgref_ins_ural_repair"]] call _fnc_saveToTemplate;
+["vehiclesFuelTrucks", ["rhsgref_ins_kraz255b1_fuel"]] call _fnc_saveToTemplate;
+["vehiclesMedical", ["rhsgref_ins_gaz66_ap2"]] call _fnc_saveToTemplate;
+
+["vehiclesLightAPCs", ["rhsgref_BRDM2_ins","rhsgref_BRDM2_ins", "rhsgref_BRDM2UM_ins", "rhsgref_BRDM2_HQ_ins"]] call _fnc_saveToTemplate;            //this line determines light APCs
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:48]: Drop the unarmed one (UM) or put it into trucks or light unarmed. (very light) armour isn't relevant unless the vehicle itself is a threat. BRDMs can go in light armed instead but that's optional depending on what spaces you need to fill.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 5]
```diff
@@ -0,0 +1,1028 @@
...
+["vehiclesAPCs", ["rhsgref_ins_btr60","rhsgref_ins_btr70"]] call _fnc_saveToTemplate;
+["vehiclesIFVs", ["rhsgref_ins_bmd2","rhsgref_ins_bmd1p","rhsgref_ins_bmd1","rhsgref_ins_bmp2k","rhsgref_ins_bmp2d","rhsgref_ins_bmp2e","rhsgref_ins_bmp2","rhsgref_ins_bmp1p","rhsgref_ins_bmp1k","rhsgref_ins_bmp1d","rhsgref_ins_bmp1"]] call _fnc_saveToTemplate;                //this line determines IFVs
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:50]: I'd bump the BMDs down to APC. They might have a lot of firepower but they seem a lot easier to kill than the BMPs, and the cargo guys ride in the open.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf (hunk 2 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf]: I moved the rocket hips up to light attack, and added the AMTSh for varerity

Comment (ante185) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf]: for clarity, when you talk about setting it as a cheap attack heli are you refering to the mtv3s or the Mi24's?

Comment (jaj22) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf]: The Mi-24P specifically. You can put it in attack helis but I don't think it's worth the full value.

Comment (ante185) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf]: Amended

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 6]
```diff
@@ -0,0 +1,1038 @@
...
+["flagMarkerType", "a3a_flag_chdkz"] call _fnc_saveToTemplate;
+/////////////////////
+///  Identities   ///
+/////////////////////
+
+["faces", ["LivonianHead_6", "WhiteHead_01", "WhiteHead_02", "WhiteHead_03", "WhiteHead_04",
+"WhiteHead_06", "WhiteHead_07", "WhiteHead_08", "WhiteHead_10", "WhiteHead_11",
+"WhiteHead_13", "WhiteHead_15", "WhiteHead_16", "WhiteHead_17", "WhiteHead_18",
+"WhiteHead_19", "WhiteHead_20", "WhiteHead_21"]] call _fnc_saveToTemplate;
+["voices", ["RHS_Male01RUS", "RHS_Male02RUS", "RHS_Male03RUS", "RHS_Male04RUS", "RHS_Male05RUS","Male01RUS","Male02RUS","Male03RUS"]] call _fnc_saveToTemplate;
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:19]: Stick to RHS or Vanilla Voices.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 7]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_PG7V_mag", "rhs_rpg7_PG7VL_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_PG7VM_mag", "rhs_rpg7_PG7VL_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_PG7VL_mag", "rhs_rpg7_type69_airburst_mag", "rhs_rpg7_OG7V_mag"], [], ""]
+]];
+_loadoutData set ["heavyATLaunchers", [
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_PG7VR_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_PG7VS_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "rhs_acc_pgo7v3",["rhs_rpg7_TBG7V_mag"], [], ""]
+]];
+_loadoutData set ["missileATLaunchers", ["launch_O_Oorona_green_F"]];
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:122]: No Vanilla Weapons in RHS.

That is also Spelled wrong "launch_O_Oorona_green_F" should be "launch_O_Vorona_green_F"

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 8]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_aks74u", "rhs_acc_pbs4", "", "", ["rhs_30Rnd_545x39_7U1_AK"], [], ""],
+["rhs_weap_akms", "rhs_acc_pbs1", "rhs_acc_2dpZenit", "", ["rhs_30Rnd_762x39mm_U"], [], ""],
+["rhs_weap_m92", "rhs_acc_pbs1", "", "", ["rhs_30Rnd_762x39mm_U"], [], ""]
+]];
+_militiaLoadoutData set ["grenadeLaunchers", [
+["rhs_weap_akm_gp25", "rhs_acc_dtkakm", "", "", ["rhs_30Rnd_762x39mm", "rhs_30Rnd_762x39mm_tracer"], ["rhs_VOG25", "rhs_VOG25", "rhs_VOG25P", "rhs_VG40OP_white"], ""],
+["rhs_weap_ak74_gp25", "rhs_acc_dtk1983", "", "", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], ["rhs_VOG25", "rhs_VOG25", "rhs_VOG25P", "rhs_VG40OP_white"], ""],
+["rhs_weap_aks74_gp25", "rhs_acc_dtk1983", "", "", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], ["rhs_VOG25", "rhs_VOG25", "rhs_VOG25P", "rhs_VG40OP_white"], ""]
+]];
+//_militiaLoadoutData set ["SMGs", []];
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:333]: just remove SMGs, dont add dead comments

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 9]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_rpk74m", "rhs_acc_dtkrpk", "rhs_acc_2dpZenit", "rhs_acc_1p78", ["rhs_60Rnd_545X39_7N10_AK", "rhs_60Rnd_545X39_AK_Green"], [], ""],
+["rhs_weap_rpk74m", "rhs_acc_dtkrpk", "rhs_acc_2dpZenit", "rhs_acc_1p29", ["rhs_60Rnd_545X39_7N10_AK", "rhs_60Rnd_545X39_AK_Green"], [], ""]
+]];
+_militaryLoadoutData set ["marksmanRifles", [
+["rhs_weap_ak103", "rhs_acc_dtk", "", "rhs_acc_pso1m2", ["rhs_30Rnd_762x39mm", "rhs_30Rnd_762x39mm_tracer"], [], ""],
+["rhs_weap_ak74n", "rhs_acc_dtk", "rhs_acc_2dpZenit", "rhs_acc_pso1m2", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], [], ""]
+]];
+_militaryLoadoutData set ["sniperRifles", [
+["rhs_weap_svdp", "", "", "rhs_acc_pso1m21", ["rhs_10Rnd_762x54mmR_7N1"], [], ""],
+["rhs_weap_m76", "", "", "rhs_acc_pso1m2", ["rhsgref_10Rnd_792x57_m76", "rhssaf_10Rnd_792x57_m76_tracer"], [], ""]
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:261]: There is no need to add a SAF Dependecy.

"rhssaf_10Rnd_792x57_m76_tracer" is not needed.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 10]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_ak74n", "rhs_acc_dtk1983", "rhs_acc_2dpZenit", "", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], [], ""],
+["rhs_weap_aks74n_2", "rhs_acc_dtk1983", "rhs_acc_2dpZenit", "", ["rhs_30Rnd_545x39_7N6M_plum_AK","rhs_30Rnd_545x39_7N10_plum_AK", "rhs_30Rnd_545x39_AK_plum_green"], [], ""],
+["rhs_weap_ak74n_2", "rhs_acc_dtk1983", "rhs_acc_2dpZenit", "", ["rhs_30Rnd_545x39_7N6M_plum_AK","rhs_30Rnd_545x39_7N10_plum_AK", "rhs_30Rnd_545x39_AK_plum_green"], [], ""]
+]];
+_militaryLoadoutData set ["carbines", [
+["rhs_weap_aks74un", "rhs_acc_pgs64_74u", "", "", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], [], ""],
+["rhs_weap_aks74n", "rhs_acc_dtk1983", "", "", ["rhs_30Rnd_545x39_7N6M_AK","rhs_30Rnd_545x39_7N10_AK", "rhs_30Rnd_545x39_AK_green"], [], ""]
+]];
+_militaryLoadoutData set ["saboteurCarbines", [
+["rhs_weap_aks74un", "rhs_acc_pbs4", "", "rhs_acc_ekp8_18", ["rhs_30Rnd_545x39_7U1_AK"], [], ""],
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:234]: Optic: rhs_acc_ekp8_18 is incompatible with rhs_weap_m92 | Comaptible optics: []

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 11]
```diff
@@ -0,0 +1,1038 @@
...
+]];
+_sfLoadoutData set ["SMGs", [
+["rhs_weap_pp2000", "", "", "rhs_acc_1p87", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_rakursPM", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_ekp8_18", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_okp7_picatinny", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "", ["rhs_mag_9x19mm_7n31_44"], [], ""]
+]];
+_sfLoadoutData set ["carbines", [
+["rhs_weap_m92", "rhs_acc_pbs1", "", "rhs_acc_1p78", ["rhs_30Rnd_762x39mm_U"], [], ""],
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:406]: Optic: rhs_acc_1p78 is incompatible with rhs_weap_m92 | Comaptible optics: []

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 12]
```diff
@@ -0,0 +1,1038 @@
...
+//];
+
+
+////////////////////////////////
+//    Militia Loadout Data    //
+////////////////////////////////
+
+private _militiaLoadoutData = _militaryLoadoutData call _fnc_copyLoadoutData;
+//_militiaLoadoutData set ["uniforms", []];
+//_militiaLoadoutData set ["vests", []];
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:273]: No need for Dead Comments.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 13]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_rpg7", "", "", "",["rhs_rpg7_PG7VL_mag", "rhs_rpg7_type69_airburst_mag", "rhs_rpg7_OG7V_mag"], [], ""]]];
+_militiaLoadoutData set ["heavyATLaunchers", [
+["rhs_weap_rpg7", "", "", "",["rhs_rpg7_PG7VR_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "",["rhs_rpg7_PG7VS_mag"], [], ""],
+["rhs_weap_rpg7", "", "", "",["rhs_rpg7_TBG7V_mag"], [], ""]
+]];
+
+
+_militiaLoadoutData set ["slRifles", [
+["rhs_weap_akm", "rhs_acc_dtk", "rhs_acc_2dpZenit", "", ["rhs_30Rnd_762x39mm", "rhs_30Rnd_762x39mm_tracer"], [], ""],
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:297]: Muzzle: rhs_acc_dtk is incompatible with rhs_weap_akm | Comaptible muzzles: ["rhs_acc_dtk1l","rhs_acc_dtkakm","rhs_acc_pbs1"]

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 14]
```diff
@@ -0,0 +1,1038 @@
...
+//    Military Loadout Data    //
+/////////////////////////////////
+
+private _militaryLoadoutData = _loadoutData call _fnc_copyLoadoutData;
+_militaryLoadoutData set ["uniforms", ["rhsgref_uniform_reed","rhsgref_uniform_reed","rhs_uniform_gorka_r_y","rhs_uniform_gorka_r_g","rhsgref_uniform_vsr"]];
+_militaryLoadoutData set ["vests", ["rhs_6b3_AK_3","rhs_6b3_AK_2","rhs_6b3_AK","rhs_6b3","rhs_6b2_AK","rhs_6b2_chicom","rhs_6b2_lifchik","rhs_6b2_RPK","rhs_6b2_SVD","rhs_6sh92_digi","rhs_6sh92_digi_radio","rhs_6sh92","rhs_6sh92_radio","V_TacVest_blk","V_BandollierB_rgr","V_Chestrig_rgr"]];
+_militaryLoadoutData set ["glVests", ["rhs_6sh92_digi_vog","rhs_6sh92_vog","rhs_6b3_VOG","rhs_6b3_VOG_2"]];
+_militaryLoadoutData set ["slVests", ["rhs_6sh92_digi_vog_headset","rhs_6sh92_vog_headset","rhs_6b3_VOG","rhs_6b3_VOG_2"]];
+_militaryLoadoutData set ["backpacks", []];
+_militaryLoadoutData set ["slBackpacks", ["rhs_r148"]];
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:198]: rhs_r148 has almost no Cargo Space for Equipment.

The SL Loadout for Miltary will always be overfilled, reduce the SL Loadouts Magazines/Equipment or give a Backpack that can carry something.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 15]
```diff
@@ -0,0 +1,1038 @@
...
+_loadoutData set ["items_unarmed_extras", []];
+
+//TODO - ACE overrides for misc essentials, medical and engineer gear
+
+/////////////////////////////////
+//    Military Loadout Data    //
+/////////////////////////////////
+
+private _militaryLoadoutData = _loadoutData call _fnc_copyLoadoutData;
+_militaryLoadoutData set ["uniforms", ["rhsgref_uniform_reed","rhsgref_uniform_reed","rhs_uniform_gorka_r_y","rhs_uniform_gorka_r_g","rhsgref_uniform_vsr"]];
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:193]: to many uniforms with Different Colors, stick to just a few Uniforms with matching Colors to the rest of their Equipment.

throw the worse ones to Militia.

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 16]
```diff
@@ -0,0 +1,1038 @@
...
+["rhs_weap_svds", "rhs_acc_tgpv2", "", "rhs_acc_pso1m2", ["rhs_10Rnd_762x54mmR_7N14"], [], ""],
+["rhs_weap_svds_npz", "rhs_acc_tgpv2", "", "rhs_acc_dh520x56", ["rhs_10Rnd_762x54mmR_7N14"], [], ""]
+]];
+_sfLoadoutData set ["sniperRifles", [
+["rhs_weap_t5000", "", "", "rhs_acc_dh520x56", [], [], "rhs_acc_harris_swivel"]
+]];
+_sfLoadoutData set ["sidearms", [
+["rhs_weap_pb_6p9", "rhs_acc_6p9_suppressor", "", "", [], [], ""],
+["rhs_weap_pp2000_folded", "", "", "rhs_acc_ekp8_18", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["hgun_Rook40_F", "muzzle_snds_L", "", "", ["16Rnd_9x21_Mag"], [], ""]
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:439]: No Vanilla Items.

**Review by Lazejun — CHANGES_REQUESTED** [review]
See Comments,

Stop being Special and Order the Loadouts like all the Other Templates are.
SF, Military, Police, Militia, Crew.

Not Military, Militia, SF, Police, Crew

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf (hunk 14 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:287]: Unfortunate

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf (hunk 15 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf]: fixed, but with regret

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf [hunk 17]
```diff
@@ -0,0 +1,988 @@
...
+]];
+_sfLoadoutData set ["SMGs", [
+["rhs_weap_pp2000", "", "", "rhs_acc_1p87", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_rakursPM", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_ekp8_18", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "rhs_acc_okp7_picatinny", ["rhs_mag_9x19mm_7n31_44"], [], ""],
+["rhs_weap_pp2000", "", "", "", ["rhs_mag_9x19mm_7n31_44"], [], ""]
+]];
+_sfLoadoutData set ["carbines", [
+["rhs_weap_m92", "rhs_acc_pbs1", "", "rhs_acc_1p78", ["rhs_30Rnd_762x39mm_U"], [], ""],
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_ChDKZ.sqf:241]: Optic: rhs_acc_1p78 is incompatible with rhs_weap_m92 | Comaptible optics: []

**Comment by ante185** [comment]
#2790 and this PR has the flag textures for the map markers in different locations
Should i, while resolving the conflicts, move my texture or theirs, or do i leave all of them where there are currently despite the fact that they should be in the same place?

**Comment by jaj22** [comment]
> #2790 and this PR has the flag textures for the map markers in different locations Should i, while resolving the conflicts, move my texture or theirs, or do i leave all of them where there are currently despite the fact that they should be in the same place?

Move theirs if you like. Make sure you use git mv rather than delete/add to reduce the chances of data-spamming.

**Comment by ante185** [comment]
> 
> Move theirs if you like. Make sure you use git mv rather than delete/add to reduce the chances of data-spamming.

Done, but might have possibly screwed up in another way instead

**Comment by ante185** [comment]
As done as can be following completion of requested changes, pending re-review.

**Review by Lazejun — APPROVED** [review]

**Pull request merged**